ASPNET-CORE-MVC/Domain/Dtos/TransactionDtos/TransactionDto.cs

```diff
@@ -21,8 +21,7 @@ public TransactionDto(Transaction transaction)
         Planned = transaction.Planned;
     }
 
-    [Required]
-    public Guid Id { get; set; }
+    [Required] public Guid Id { get; set; }
```

Comment:
Could it be `public required` as in other DTOs?

Reply:
This is different costructions. Keyword "required" means that the filed must be set during initialiation, [Required] is about that field that comes from form must be set.